docs/sdkx-react-native/getting-started.mdx

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: 10
+title: Getting Started
+description: "You're 3 steps away from adding great in-app support to your React native application."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Getting Started {#getting-started}
+
+<Intro>
+
+You're 3 steps away from adding great in-app support to your React Native application.
+
+</Intro>
+
+Guide to integrating the React Native plugin for the Helpshift SDK X.
+
+## Requirements {#version-requirements}
+
+- Node v16 or above
+- Android Studio
+- Xcode 13 or above
+- Java 11 or above
+
+## Download Helpshift React native SDK {#getting-sdk}
+
+#### SDK X - In-app Customer Service {#standard-sdk}
+
```

Comment:
I think ambiguity here because you show the download button and redirect to the npm package. 
Instead, we should give the npm command to download latest package. Give Npm link for more info.

Reply:
Updated. Please find https://www.npmjs.com/package/helpshift-plugin-sdkx-react-native